sdk/python/tests/integration/feature_repos/universal/entities.py

```diff
@@ -10,5 +10,10 @@ def driver():
     )
 
 
+def driver_no_value_type():
+    # Don't specify value type in entity to force inference
+    return Entity(name="driver", description="driver id", join_key="driver_id",)
+
+
```

Comment:
Do you want to use the existing driver entity but add some params to customize it? Like:
```
def driver(value=True):
    return Entity(
        name="driver",  # The name is derived from this argument, not object name.
        value_type=ValueType.INT64 if Value else ValueType.UNKNOWN,
        description="driver id",
        join_key="driver_id",
    )
```

Reply:
Done!